pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java

```diff
@@ -92,6 +93,28 @@
         }
     }
 
+    @GET
+    @Path("/leaderBroker")
+    @ApiOperation(
+            value = "Get the service url of the leader broker in the cluster.",
+            response = String.class)
+    @ApiResponses(
+            value = {
+                    @ApiResponse(code = 401, message = "Authentication required"),
+                    @ApiResponse(code = 403, message = "This operation requires super-user access") })
+    public String getLeaderBroker() throws Exception {
+        validateSuperUserAccess();
+
+        try {
+            return pulsar().getLeaderElectionService().getCurrentLeader()
+                    .map(LeaderBroker::getServiceUrl)
+                    .orElse("None");
```

Comment:
Nice contribution!

Now we will return null if we have no data. I think It is better to keep the same, otherwise the status returned by the Broker side will be more and more.

Reply:
Thanks for your comment.
But maybe unclear semantics if we print "" on the console.

Reply:
In CMD, we can handle the case of null, but the exposed REST API should be consistent

Reply:
Thanks, i will take your advice.